### PR TITLE
PT-711 display Rescucarbs with their own layout

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: plugins/blip/chartdailyfactory.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ function chartDailyFactory(el, options) {
         'light': ' g'
       }])
       .labelBaseline(options.labelBaseline)
-      .legend(['bolus', 'carbs'])
+      .legend(['bolus', 'carbs', 'rescuecarbs'])
```
Comment (yvon-dblg) [inline plugins/blip/chartdailyfactory.js:126]: Must check "rescuecarbs" or "Rescuecarbs" for the translation.

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
Translation to review.

File: plugins/blip/chartdailyfactory.js (hunk 1 above)
Comment (franck-fourel) [reply plugins/blip/chartdailyfactory.js:126]: let's stick to 'rescuecarbs'

**Review by yvon-dblg — APPROVED** [review]

**Comment by yvon-dblg** [comment]
Missing Changelog release version + date before merge

**Pull request merged**